src/System Application/App/AI/src/Copilot/CopilotQuotaUsageType.Enum.al

```diff
@@ -18,14 +18,14 @@ enum 7785 "Copilot Quota Usage Type"
     /// </summary>
     value(0; "Generative AI Answer")
     {
-        Caption = 'Generative AI Answer';
+        Caption = 'Generative answer';
```

Comment:
Generative Answer

Reply:
UX instructions are to use lower letters